src/time_travel/patchers/basic_patcher.py

```diff
@@ -4,11 +4,13 @@
 class BasicPatcher(object):
     """Base class for patching time modules."""
     
-    def __init__(self, clock):
+    def __init__(self, clock, events_pool=None):
         """Create the patch."""
         self.clock = clock
-        self.patches = []
+        self.events_pool = events_pool
```

Comment:
Not sure this member belongs in this class. It obviously doesn't use it - why not just put it in the child class?